### PR TITLE
docs(env-checker): add faq about the extension settings related to envchecker

### PR DESCRIPTION
**Pull request by qinezh**
You could review the image from here: https://github.com/OfficeDev/TeamsFx/blob/qinezh/doc/docs/vscode-extension/envchecker-help.md#teams-toolkit-validation-settings

### CODE REVIEW

File: docs/vscode-extension/envchecker-help.md [hunk 1]
```diff
@@ -89,6 +89,19 @@ Please refer to [the guide](https://docs.microsoft.com/dotnet/core/tools/sdk-err
 
 The `global.json` file is not required, so if it doesn't contain anything other than the sdk node, you can delete the whole file.
 
+## Teams Toolkit Validation Settings
+
+There're two settings to turn on/off .NET SDK and Node.js validation, and both of them are enabled by default. You are able to uncheck the box if you don't need the dependencies validation and would like to handle it by yourself.
+
+To open your user and workspace settings, use the following VS Code menu command:
```
Comment (a1exwang) [inline docs/vscode-extension/envchecker-help.md:96]: VSCode -> Visual Studio Code
There're -> There are
don't need -> do not need (similar to the previous changes from Tim)

Comment (qinezh) [reply docs/vscode-extension/envchecker-help.md:96]: image
VS Code is copied from the official documentation 😂: https://code.visualstudio.com/docs/getstarted/settings#_creating-user-and-workspace-settings

**Review by a1exwang — APPROVED** [review]

File: docs/vscode-extension/envchecker-help.md (hunk 1 above)
Comment (qinezh) [reply docs/vscode-extension/envchecker-help.md]: Updated to keep consistent with previous updates

**Review by meifans — APPROVED** [review]
LGTM

File: docs/vscode-extension/envchecker-help.md [hunk 2]
```diff
@@ -89,6 +89,17 @@ Please refer to [the guide](https://docs.microsoft.com/dotnet/core/tools/sdk-err
 
 The `global.json` file is not required, so if it doesn't contain anything other than the sdk node, you can delete the whole file.
 
+## Teams Toolkit Validation Settings
+
+There are two settings to turn on/off .NET SDK and Node.js validation, and both of them are enabled by default. You are able to uncheck the box if you don not need the dependencies validation and would like to handle it by yourself.
```
Comment (kimizhu) [inline docs/vscode-extension/envchecker-help.md:94]: don't? or do not?

Comment (qinezh) [reply docs/vscode-extension/envchecker-help.md]: good catch, updated.

**Review by kimizhu — APPROVED** [review]

**Pull request merged**